modules/core/server/views/email-templates/email-with-container.server.view.html

```diff
@@ -0,0 +1,59 @@
+{% extends './email.server.view.html' %}
+
+{% block content %}
+
+  <!-- MODULE ROW // -->
+  <tr>
+      <td align="center" valign="top">
+          <!-- CENTERING TABLE // -->
+            <!--
+              The centering table keeps the content
+                tables centered in the emailBody table,
+                in case its width is set to 100%.
+            -->
+          <table border="0" cellpadding="0" cellspacing="0" width="100%">
```

Comment:
Is there a way to make this DRYer?
I understand we need messy tables like this for emails, but would be good to keep it limited to one spot, if possible?

May be best to open an issue for this for now, as there are a dozen of these:
<img width="759" alt="image" src="https://user-images.githubusercontent.com/54728/49942993-16e7f180-fee7-11e8-98be-cec3eff96a09.png">

Reply:
Yep, that's a refactor long due.

Basically, those tables would need to be removed to a layout and reused each time with `{% extends './layout.server.view.html' %}`

Reply:
Templates are Nunjucks templates: https://mozilla.github.io/nunjucks/

Reply:
There is an attempt for drying in this PR. See ` modules/core/server/views/email-templates/email-with-container.server.view.html`. It assumes a message and an action button at the end.